app/assets/stylesheets/application.css

```diff
@@ -17,3 +18,7 @@
 .row {
   padding-bottom: 1em;
 }
+
+.video.col-sm-2 {
```

Comment:
Rather than create this specific of a class, I prefer creating a class named something like `no-horizontal-margin` and maybe a `large-vertical-margin`. Then you can apply this style elsewhere easily. This is not a requirement, just an idea.